php-http/httplug-bundle/1.5/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "bundles": {
+        "Http\\HttplugBundle\\HttplugBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
+    },
+    "composer-scripts": {
+        "assets:install --symlink --relative %WEB_DIR%": "symfony-cmd"
```

Comment:
This is not needed as already part of framework-bundle, so it's aways installed.